torchgen/gen_executorch.py

```diff
@@ -123,10 +123,15 @@ class ComputeFunction:
 
     @method_with_native_function
     def __call__(self, f: NativeFunction) -> Optional[str]:
+        is_method_variant = False
         if not self.selector.is_root_operator(f"{f.namespace}::{f.func.name}"):
             return None
         if Variant.function not in f.variants:
-            return None
+            if Variant.method in f.variants:
+                is_method_variant = True
+            else:
+                return None
+
```

Comment:
Can we error out in this case? It took us a while to understand what's going on. We also want to error out if both method and function are in variants.